pgb_utils/pgb_utils/__init__.py

```diff
@@ -5,4 +5,8 @@
 and services.
 """
 
```

Comment:
Echo from zoom convo: Add a warning if environmental variables are not set.

Reply:
✅